devtools-ci-index.yaml

```diff
@@ -3778,6 +3873,21 @@
             ee_test_start_time: '20 6,18 * * *'
             timeout: 60m
             zabbix_enabled: false
+        - 'devtools-prebooster-test-{test_url}-{cluster}-{booster_name}':
+            test_url: openshift.io
+            prebooster_test_start_time: '5 0-23/4 * * *'
+            osio_prebooster_creds_token: 'prebooster_token_1'
+            project_name: osio-ci-boost1-preview-stage
+            booster_name: golang-health-check
+            cluster: https://api.starter-us-east-2a.openshift.com
```

Comment:
do you really want such a long string with special characters to get into job name?

Reply:
Good catch! Yes - that would make for a very awkward, if not totally invalid, URL. Thanks!